Cython/Utility/TypeConversion.c

```diff
@@ -509,7 +509,9 @@ static CYTHON_INLINE PyObject* __Pyx__PyNumber_Float(PyObject* obj) {
     if (unlikely(val == -1 && PyErr_Occurred())) {
         return NULL;
     }
+#if CYTHON_USE_PYLONG_INTERNALS
 no_error:
+#endif
```

Comment:
This silences a warning about unused `no_error` if not `CYTHON_USE_PYLONG_INTERNALS` is not defined